methcomp/blandaltman.py

```diff
@@ -115,76 +123,44 @@ def plot(self, ax: matplotlib.axes.Axes):
         ax.spines['top'].set_visible(False)
 
         # set X and Y limits
-        if self.xlim is not None:
-            ax.set_xlim(self.xlim[0], self.xlim[1])
-        if self.ylim is not None:
-            ax.set_ylim(self.ylim[0], self.ylim[1])
+        if xlim is not None:
+            ax.set_xlim(xlim[0], xlim[1])
+        if ylim is not None:
+            ax.set_ylim(ylim[0], ylim[1])
 
         # graph labels
-        ax.set_ylabel(self.y_title)
-        ax.set_xlabel(self.x_title)
-        if self.graph_title is not None:
-            ax.set_title(self.graph_title)
+        ax.set_ylabel(y_title)
+        ax.set_xlabel(x_title)
+        if graph_title is not None:
+            ax.set_title(graph_title)
```

Comment:
Check for None is not needed, it will be set to '' by matplotlib
```python
ax.set(
   xlabel=x_title,
   ylabel=y_title,
   title=graph_title)
```

Reply:
thanks, resolved